c++/conductances/soplata/reticular/Kd.hpp

```diff
@@ -0,0 +1,56 @@
+// _  _ ____ _    ____ ___ _
+//  \/  |  | |    |  |  |  |
+// _/\_ |__| |___ |__|  |  |___
+//
+// delayed rectifier potassium conductance
+// Soplata AE, McCarthy MM, Sherfey J, Lee S, Purdon PL, Brown EN, et al.
+// Thalamocortical control of propofol phase-amplitude coupling. PLOS
+// Computational Biology. 2017;13: e1005879. doi:10.1371/journal.pcbi.1005879
+
+#ifndef KD
+#define KD
+#include "conductance.hpp"
+
+//inherit conductance class spec
+class Kd: public conductance {
+
+public:
+
+    //specify both gbar and erev and initial conditions
+    Kd(double g_, double E_, double m_)
+    {
+        gbar = g_;
+        E = E_;
+        m = m_;
+
+        // defaults
+        if (isnan(gbar)) { gbar = 0; }
+        if (isnan (m)) { m = 0; }
+        if (isnan (E)) { E = -80; }
+
+        p = 4;
+
+        // allow this channel to be approximated
+        approx_m = 1;
```

Comment:
it looks like this channel has extra dynamic variables, does it actually work if you allow approximations? if so, go for it. 

otherwise set this flag to 0.